bot/exts/evergreen/githubinfo.py

```diff
@@ -33,6 +193,71 @@ async def github_group(self, ctx: commands.Context) -> None:
         if ctx.invoked_subcommand is None:
             await invoke_help_command(ctx)
 
+    @commands.Cog.listener()
+    async def on_message(self, message: discord.Message) -> None:
+        """
+        Automatic issue linking.
+
+        Listener to retrieve issue(s) from a GitHub repository using automatic linking if matching <org>/<repo>#<issue>.
+        """
+        # Ignore bots
+        if message.author.bot:
+            return
+
+        issues = [
+            FoundIssue(*match.group("org", "repo", "number"))
+            for match in AUTOMATIC_REGEX.finditer(self.remove_codeblocks(message.content))
+        ]
+        links = []
+
+        if issues:
+            # Block this from working in DMs
+            if not message.guild:
+                await message.channel.send(
```

Comment:
Get rid of the response here, no one is expecting this to work in dms IMO.
![image](https://user-images.githubusercontent.com/71233171/131932699-ff4b79d4-ef88-4ea6-882a-d8635b600f4e.png)